examples/yoti_example_flask/templates/profile.html

```diff
@@ -60,10 +60,10 @@ <h3><a href="/">Home</a></h3>
             </tr>
         {% endif %}
 
-        {% if age_verified %}
+        {% if is_age_verified is defined %}
```

Comment:
Flask used `Jinja`, which uses slightly different syntax to what we're using in the Django project (http://jinja.pocoo.org/docs/2.10/templates/#defined)